### ISSUE TITLE
Request: Disassembly Instructions

### ISSUE THREAD

**Issue by carlossless** [nox-x/TG3442DE-Teardown#5]
Hey, thank you for all the efforts directed at documenting this device!

I am interested in possibly making a rack-mountable shell for this thing, but I am a bit confused by how is it supposed to be disassembled. Removing all 4 visible screws does not cause it to fall apart, and it won't budge even with a little applied force in all directions and cracks.

I would love to keep the original shell intact and not destroy it utterly, any hints?

**Comment by nox-x** [comment]
I've successfully used an old (plastic) credit card divided in 3 stripes (credit card plastic seems strong enough but not to hard for damaging the material). With theses stripes I was able to disjoin each couple of hooks starting at the bottom side.
It's a bit tricky because the assembler also used tongue and groove joints in addition to the small hooks.
I would recommend to first disjoin the bottom, then the back & front sides and at least the top side 
Hopefully the attached photos from inside the case can make it easier:

![back](https://user-images.githubusercontent.com/19293267/142776855-2a522964-b618-49f8-8b37-348648ca3c65.jpg)
![bottom](https://user-images.githubusercontent.com/19293267/142776856-e961f6d7-0733-47b7-bf30-1f1d8848bda4.jpg)
![front](https://user-images.githubusercontent.com/19293267/142776857-0bc2df2a-cd27-40e1-94f0-0893c705c648.jpg)
![top](https://user-images.githubusercontent.com/19293267/142776858-977c3e1a-ccc8-4976-b879-7326b6687c44.jpg)
